google-oauth-client-jetty/src/main/java/com/google/api/client/extensions/jetty/auth/oauth2/LocalServerReceiver.java

```diff
@@ -69,14 +71,32 @@
   private final String host;
 
   /**
-   * Constructor that starts the server on {@code "localhost"} selects an unused port.
+   * URL to an HTML page to be shown (via redirect) after successful login. If null, a canned
+   * default landing page will be shown (via direct response).
+   */
+  private String successLandingPageUrl;
+
+  /**
+   * URL to an HTML page to be shown (via redirect) after failed login. If null, a canned
+   * default landing page will be shown (via direct response).
+   */
+  private String failureLandingPageUrl;
+
+  /**
+   * Constructor that starts the server on {@link #LOCALHOST} and an unused port.
    *
    * <p>
    * Use {@link Builder} if you need to specify any of the optional parameters.
    * </p>
    */
   public LocalServerReceiver() {
-    this("localhost", -1);
+    this(LOCALHOST, -1);
+  }
+
+  public LocalServerReceiver(String successLandingPageUrl, String failureLandingPageUrl) {
```

Comment:
Instead of having another public constructor, it looks like these options should be added to the Builder instead.